x-pack/plugins/cases/public/components/user_actions/comment/registered_attachments.tsx

```diff
@@ -128,10 +133,24 @@ export const createRegisteredAttachmentUserActionBuilder = <
         timelineAvatar: attachmentViewObject.timelineAvatar,
         actions: (
           <UserActionContentToolbar id={comment.id}>
-            {attachmentViewObject.actions}
+            {visiblePrimaryActions.map((action) => (
```

Comment:
Ah nice that you ended up going with the visible and non visible actions 👍

Reply:
🙂 I followed the guidelines of the EUI. They do the same in the tables.